pymc/tests/test_util.py

```diff
@@ -174,9 +172,36 @@ def test_find_optim_prior():
         init_guess={"mu": 170, "sigma": 3},
         fixed_params={"nu": 7},
     )
-    np.testing.assert_allclose(np.asarray(opt_params.values()), np.array([0.25, 0.06343503]))
+    assert "nu" in opt_params
+    np.testing.assert_allclose(
+        list(opt_params.values()), np.array([0.24995405785756986, 0.06343501657095188, 7])
+    )
 
-    with pytest.raises(ValueError, match="parameters, but you provided only"):
+    # Student not deterministic but without warning
+    with pytest.warns(None) as record:
+        pm.find_optim_prior(
+            pm.StudentT,
+            lower=0,
+            upper=1,
+            mass=MASS,
+            init_guess={"mu": 5, "sigma": 2, "nu": 7},
+        )
+    assert len(record) == 0
+
+    # Exponential without warning
+    with pytest.warns(None) as record:
+        opt_params = pm.find_optim_prior(
+            pm.Exponential, lower=0, upper=1, mass=MASS, init_guess={"lam": 1}
+        )
+    assert len(record) == 0
```

Comment:
I would add these to all that are expected to work.

Reply:
Done